portfolio/src/main/webapp/snake.js

```diff
@@ -0,0 +1,127 @@
+const cvs = document.getElementById("snake");
```

Comment:
is this snake game part of the curriculum? i'm not sure whether it makes sense to review

Reply:
No it is not part of the curriculum so You don't have to worry about it